Vungle Sample App Swift/Vungle Sample App Swift/ViewController.swift

```diff
@@ -226,30 +156,96 @@ class ViewController: UIViewController, VungleSDKDelegate {
     
     private func showAdForPlacement03 () {
         // Play a Vungle ad (with options). Dictionary to set custom ad options.
-        let options: NSDictionary = NSDictionary(dictionary: [VunglePlayAdOptionKeyUser: "test_user_id",
-                                                              VunglePlayAdOptionKeyIncentivizedAlertBodyText: "If the video isn't completed you won't get your reward! Are you sure you want to close early?",
-                                                              VunglePlayAdOptionKeyIncentivizedAlertCloseButtonText: "Close",
-                                                              VunglePlayAdOptionKeyIncentivizedAlertContinueButtonText: "Keep Watching",
-                                                              VunglePlayAdOptionKeyIncentivizedAlertTitleText: "Careful!"])
+        let options = [VunglePlayAdOptionKeyUser: "test_user_id",
+                       VunglePlayAdOptionKeyIncentivizedAlertBodyText: "If the video isn't completed you won't get your reward! Are you sure you want to close early?",
+                       VunglePlayAdOptionKeyIncentivizedAlertCloseButtonText: "Close",
+                       VunglePlayAdOptionKeyIncentivizedAlertContinueButtonText: "Keep Watching",
+                       VunglePlayAdOptionKeyIncentivizedAlertTitleText: "Careful!"]
```

Comment:
I wasn't sure why this was a dictionary wrapped in a dictionary, which was then cast to `[AnyHashable : Any]` so I unwrapped it.